validation/src/test/scala/hmda/validation/rules/lar/macro/Q073Spec.scala

```diff
@@ -0,0 +1,52 @@
+package hmda.validation.rules.lar.`macro`
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+
+class Q073Spec extends MacroSpec {
+
+  val numOfLoanApplications = config.getInt("hmda.validation.macro.Q073.numOfLoanApplications")
+  val multiplier = config.getDouble("hmda.validation.macro.Q073.numOfLarsMultiplier")
+
+  def irrelevantLar(lar: LoanApplicationRegister) = {
+    val relevantLoan = lar.loan.copy(propertyType = 2, loanType = 2, purpose = 1)
+    lar.copy(purchaserType = 0, actionTakenType = 1, loan = relevantLoan)
+  }
+  def relevantLar(lar: LoanApplicationRegister) = {
+    val relevantLoan = lar.loan.copy(propertyType = 2, loanType = 2, purpose = 1)
+    lar.copy(purchaserType = 1, actionTakenType = 1, loan = relevantLoan)
+  }
+
+  val belowThreshold = numOfLoanApplications - 1
+  val overThreshold = numOfLoanApplications + 1
+
+  property(s"be valid if fewer than $numOfLoanApplications specified loans") {
+    val lars = larNGen(belowThreshold).sample.get
+    val validLarSource = newLarSource(lars, belowThreshold, relevantLar, irrelevantLar)
+    validLarSource.mustPass
+  }
+
+  property(s"be valid if more than $numOfLoanApplications specified loans and count > $multiplier * sold") {
+    val numOfRelevantLars = (overThreshold * multiplier).toInt + 1
```

Comment:
We don't have to use `overThreshold`, since the rule says "250 or greater". I think just using `threshold` would be a better test that we've covered the edge case.